owner/manager.go

```diff
@@ -172,7 +173,21 @@ func (m *ownerManager) CampaignOwner(ctx context.Context) error {
 	return nil
 }
 
+func recoverInOwner(funcName string, quit bool) {
+	if r := recover(); r != nil {
+		buf := util.GetStack()
+		log.Errorf("%s, %v, %s", funcName, r, buf)
+		metrics.PanicCounter.WithLabelValues(metrics.LabelDDLOwner).Inc()
+		if quit {
```

Comment:
How about remove the quit code block which would make it a real recover function?